gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileAttributesTest.java

```diff
@@ -29,14 +31,13 @@
 
   private static final byte[] HAPPY = "(✿◕ ‿◕ )ノ".getBytes(UTF_8);
 
-  @Rule
-  public final AppEngineRule appEngineRule = new AppEngineRule();
 
   private Path path;
   private Path dir;
 
   @Before
   public void before() throws Exception {
+    CloudStorageFileSystemProvider.setGCloudOptions(LocalGcsHelper.options());
```

Comment:
I'm sorry I replied before I saw JP's reply here. Arie, does what he had to say change how you feel?